docs/guides/reproducible-build-verification.md

```diff
@@ -0,0 +1,118 @@
+# Reproducible build verification
+
+[Reproducible builds](https://reproducible-builds.org/) are crucial from both
+security and open-source perspectives because they allow anyone to verify that
+the compiled binary of a software package truly matches the original source
+code. This ensures that no tampering, such as inserting malicious code during
+the build process, has occurred.
+
+The most obvious and undisputable way of verifying build reproduction is
+comparing `sha256` or `md5` hashes of two given binaries. There are, however,
+corner cases where this is not an ideal approach - such as when the same
+binary is signed with a different key.
+
+## Romscope
+
+To provide a more comprehensive way of comparing two Dasharo binaries which
```

Comment:
That is not the real reason. There is no need for more comprehensive approach, if the build from source is exactly the same as we publish as a release.

The main reason is that we embed signatures inside of the firmware binary. The wiki you link to has a page on exactly this problem and how it can be approached: https://reproducible-builds.org/docs/embedded-signatures/

Reply:
[Fixed](https://github.com/Dasharo/docs/pull/921/commits/936c81929769481e9bfc9db9e079a9d9c719569c#diff-e4434d45287f1a1685277fe23157c8a8613fe7209c8994b241a9c310469cce45R16)